### DISCUSSION TITLE
Repository access query for app

### DISCUSSION THREAD

**Discussion by TimTaylor** [r-universe-org/help#407] [General]
Currently investigating setting up a universe and I'm trying to understand why the r-universe app needs enabling for all repositories and not just  `<username>.r-universe.dev` on GitHub?.

For reference this is mentioned in the second bullet within https://github.com/r-universe-org/help/blob/master/README.md#how-to-setup-your-personal-universe

> 1. Create a repository called `<username>.r-universe.dev` on the GitHub account for `username`, for example: https://github.com/maelle/maelle.r-universe.dev. The repository must contain a file called [packages.json](https://github.com/maelle/maelle.r-universe.dev/blob/main/packages.json) in the standard format, defining at least the `package` name and git `url` for the packages you want to include.
>
> 2. Install the [r-universe app](https://github.com/apps/r-universe/installations/new) on the GitHub account that you want to enable. Choose __enable for all repositories__ when asked.

**Comment by jeroen** [comment]
It is not strictly required, but it will enable r-universe to set a ✅  or ❌  commit-status on the commit when your package is being built and when it has been deployed. 

<img width="744" alt="Screenshot 2024-05-14 at 3 55 00 PM" src="https://github.com/r-universe-org/help/assets/216319/c97b7376-9f1b-4995-a870-e8baf90a2b8a">


The github-app only asks for commit-status permissions so it can't do any harm.
